docs/Drawer.md

```diff
@@ -0,0 +1,22 @@
+# Drawer
+The Drawer component is a wrapper around the Material UI Drawer that adds specific PX Blue functionality and styling. It is used to organize content (typically navigation links) in a collapsible side panel. The PX Blue Drawer includes helper components for `DrawerHeader`, `DrawerSubheader`, `DrawerBody`, `DrawerNavGroup`, `DrawerFooter`, and `DrawerLayout` to help organize the content.
+
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 200px" alt="Nested Drawer" src="./images/drawer.png">
+</div>
+
+## DrawerHeader
+The `DrawerHeader` contains the content at the top of the `Drawer`. By default, it renders multiple lines of text in the PX Blue style.
+    
+### DrawerHeader API
+
+<div style="overflow: auto;">
+
+| Prop Name         | Description                                    | Type              | Required | Default                      |
+|-------------------|------------------------------------------------|-------------------|----------|------------------------------|
+| icon              | A component to render an icon                  | `ng-content`      | no       |                              |
```

Comment:
Refer to Huayun's update for how we want to show transcluded/ng-content elements in the documentation...she split them into their own section separate from the API table. Also, watch your labels...in the Angular docs, we're calling the first column Attributes I believe.